dds_ports/auto.py

```diff
@@ -53,6 +62,14 @@ async def prepare_sdist(self) -> AsyncIterator[Path]:
             yield clone
 
 
+class SimpleGitHubAdaptingPort(SimpleGitAdaptingPort):
+    def __init__(self, *, owner: str, repo: str, **kwargs):
```

Comment:
This loses type annotations on the base class constructor into the `kwargs`. I prefer to enumerate all of the base class arguments again, which will then help diagnose problems and refactoring down the line.

(This might not be problem if using a `dataclass`, though? I haven't checked.)